contracts/interfaces/IPawnshop.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+interface IPawnshop {
+    // Emitteed when a borrower calls borrow
+    event LendingRegistered (uint256 lendingId, address asset, uint256 amount);
+    // Emitted when the last lender lends (full amount now lent).
+    event FullyFunded (uint256 lendingId);
+    // Emitted when the review-time of a lending expired with insufficient funding.
+    event InsufficientFunding (uint256 lendingId);
+    // Emitted when an asset is available to purchase
+    event ForSale (uint256 lendingId);
+    // Emitted when an asset is sold
+    event Sold(address buyer, uint256 lendingId, uint256 amount);
+
+    // Set the interest rate in the contract. Used in future lendings.
+    // @Admin? TODO
```

Comment:
We could potentially split off Admin functions to a separate interface. Making it clearer which functions are for users